stdlib/pervasives.gr

```diff
@@ -246,6 +247,17 @@ provide { (/) }
  */
 provide { (%) }
 
+/**
+ * Computes the exponentiation of the given base and power.
+ *
+ * @param base: The base number
+ * @param power: The exponent number
+ * @returns The base raised to the given power
+ *
+ * @since v0.6.0
+ */
```

Comment:
```suggestion
```
These are pulled from the original definition location with #1687 (which also combined these separate provides, so we'll want to rebase and update when it lands shortly

Reply:
Will do, uhm one question though on the number libary how do we handle the versioning given its in the number libary as pow since `v0.5.x` but only in the runtime and pervasive since `v0.6.0`? do we just go with since `v0.5.x` x being the version it was implemented in I don't know off the top of my head and then do we just detail it having moved around in the history section?

Reply:
Probably add a `@history` entry saying it existed in `number.gr` in whatever version it was added.